README.md

```diff
@@ -5,16 +5,23 @@ Application for demonstrating surface blending with ros
 
 Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C3%B6del
 
+### 3rd Party Dependencies
+==============
+- libqd-dev
+- [wstool](http://wiki.ros.org/wstool)
```

Comment:
Ok. Then I suggest just adding the `sudo apt-get install ..` line to the installation instructions below. Just before using `wstool`.